### PR TITLE
Yield new message class in consumer and force api.version.request config value

### PR DESCRIPTION
**Pull request by corvuscrypto**
Changes as per discussion this afternoon regarding getting the timestamps for Kafka events generated. Now all messages yielded or loaded will be of confluent_kafka_helpers.Message type.

### CODE REVIEW

File: confluent_kafka_helpers/message.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "datetime", "topic"
+    ]
+
+    def __init__(self, kafka_message):
+        self.key = kafka_message.key()
+        self.partition = kafka_message.partition()
+        self.offset = kafka_message.offset()
+        self.topic = kafka_message.topic()
+
+        timestamp_type, timestamp = kafka_message.timestamp()
```
Comment (simonwahlgren) [inline confluent_kafka_helpers/message.py]: Please move the timestamp logic into it's own method so we more easily can test it

Comment (corvuscrypto) [reply confluent_kafka_helpers/message.py]: KK

File: confluent_kafka_helpers/message.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+def kafka_timestamp_to_datetime(timestamp):
+    return datetime.datetime.utcfromtimestamp(
+        timestamp / 1000.0
+    ) if timestamp is not None else None
+
+
+def extract_timestamp_from_message(kafka_message):
+        timestamp_type, timestamp = kafka_message.timestamp()
```
Comment (simonwahlgren) [inline confluent_kafka_helpers/message.py]: Double indentation?

Comment (corvuscrypto) [reply confluent_kafka_helpers/message.py]: do you not double indent?

**Pull request merged**